rhttpc-client/src/main/scala/rhttpc/client/proxy/ReliableProxy.scala

```diff
@@ -44,8 +44,8 @@ class ReliableProxy[Req, Resp](subscriberForConsumer: ActorRef => Subscriber[Cor
     import context.dispatcher
 
     override def receive: Receive = {
-      case DelayedMessage(content, delay, attempt) =>
-        handleRequest(Request(content.asInstanceOf[Correlated[Req]], attempt, delay))
+      case DelayedMessage(content, delay, attempt, date) =>
```

Comment:
Please use full name. `date` can be tricky thing - it can be date of this attemp, first attempt, response etc.